duckduckgo/macos-browser
# Fix issue not setting maintenance date

Task/Issue URL: https://app.asana.com/0/1203581873609357/1205548216861926/f

**Description**:
Fix issue where maintenance date was not set on scan for removed profiles

**Steps to test this PR**:
1. Do the steps to find a profile during scan
2. Force extracted profiles to return an empty array to simulate removed profiles
3. Check if the date to run was set
